Source/Widgets/CesiumInspector/Cesium3DTilesInspector.js

```diff
@@ -0,0 +1,364 @@
+        var stats = document.createElement('div');
+        stats.setAttribute('data-bind', 'html: statsText, css: {"cesium-cesiumInspector-show" : showStats, "cesium-cesiumInspector-hide" : !showStats}');
+        stats.setAttribute('style', 'font-size: 11px');
+        loggingPanel.contents.appendChild(stats);
+        loggingPanel.contents.appendChild(makeCheckbox('showPickStats', 'Pick Stats'));
+        var pickStats = document.createElement('div');
+        pickStats.setAttribute('data-bind', 'html: pickStatsText, css: {"cesium-cesiumInspector-show" : showPickStats, "cesium-cesiumInspector-hide" : !showPickStats}');
+        pickStats.setAttribute('style', 'font-size: 11px');
+        loggingPanel.contents.appendChild(pickStats);
+        knockout.applyBindings(viewModel, loggingPanel.contents);
+
+        stylePanel.contents.appendChild(document.createTextNode('Color Blend Mode: '));
+        var blendDropdown = document.createElement('select');
+        blendDropdown.setAttribute('data-bind', 'options: _colorBlendModes, ' +
+                                           'optionsText: "text", ' +
+                                           'optionsValue: "val", ' +
+                                           'value: colorBlendMode');
+        stylePanel.contents.appendChild(blendDropdown);
+
+
+        var styleEditor = document.createElement('textarea');
+        styleEditor.setAttribute('wrap', 'soft');
+        styleEditor.setAttribute('data-bind', 'valueUpdate: "keyup", value: _styleString, event: { keypress: _checkCompile }');
+        stylePanel.contents.className = 'cesium-cesiumInspector-styleEditor';
+        stylePanel.contents.appendChild(styleEditor);
+        var closeStylesBtn = makeButton('_compileStyle', 'Compile (Ctrl+Enter)');
+        stylePanel.contents.appendChild(closeStylesBtn);
+        var errorBox = document.createElement('div');
+        errorBox.className = 'cesium-cesiumInspector-error';
+        errorBox.setAttribute('data-bind', 'text: _editorError');
+        stylePanel.contents.appendChild(errorBox);
+        knockout.applyBindings(viewModel, stylePanel.contents);
+    }
+
+    defineProperties(Cesium3DTilesInspector.prototype, {
+        /**
+         * Gets the parent container.
+         * @memberof Cesium3DTilesInspector.prototype
+         *
+         * @type {Element}
+         */
+        container : {
+            get : function() {
+                return this._container;
+            }
+        },
+
+        /**
+         * Gets the view model.
+         * @memberof Cesium3DTilesInspector.prototype
+         *
+         * @type {Cesium3DTilesInspectorViewModel}
+         */
+        viewModel : {
+            get : function() {
+                return this._viewModel;
+            }
+        }
+    });
+
+    /**
+     * Sets the callback function to call on tileset load
+     *
+     * @param {Function} callback the callback
```

Comment:
Actually removing these as they are never used. It doesn't make sense to register callbacks with the inspector when you can just register them on the tileset itself.